packages/client/src/client.ts

```diff
@@ -1765,18 +1776,21 @@ export class SkipClient {
       }
       return calculateFee(Math.ceil(parseFloat(estimatedGasAmount)), gasPrice);
     });
-    const feeBalance = await this.balances({
-      chains: {
-        [chainID]: {
-          address: signerAddress,
-          denoms: feeAssets.map((asset) => asset.denom),
+    const feeBalance =
+      SkipClient.skipBalances ??
```

Comment:
we should check the `feeAssets` balances in `.skipBalances` instead of check if the `.skipBalances` data is available